spec/support/suspenders.rb

```diff
@@ -12,9 +12,10 @@ def create_tmp_directory
   def run_suspenders(arguments = nil)
     Dir.chdir(tmp_path) do
       Bundler.with_clean_env do
-        ENV['TESTING'] = '1'
-
-        %x(#{suspenders_bin} #{APP_NAME} #{arguments})
+        `
```

Comment:
Right. I just now noticed the `with_clean_env` call to `Bundler`. Calling `add_fakes_to_path` after that seems to work.